app/config/permissions.neon

```diff
@@ -130,7 +137,7 @@ permissions:
 
     - allow: true
       resource: group
-      role: supervisor
+      role: supervisor-student
       actions:
           - supervise
```

Comment:
This is a really bad privilege name. What can a user with the "supervise" privilege do?

Reply:
Well, this is not mine and I did not change it. The meaning is that the user may be assigned as a supervisor of a group. I can rename it to "superviseGroup", or if you have better suggestions ...

Reply:
I know, it's even possible that I wrote it :) How about "becomeSupervisor"?